### PR TITLE
Remove unused credentials

### PR DESCRIPTION
**Pull request by MancunianSam**

### CODE REVIEW

File: terraform/modules/jenkins/templates/assume_role_policy.json.tpl [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Principal": {
+        "AWS": "${role_arn}"
+      },
+      "Action": "sts:AssumeRole"
+    }
+  ]
+}
```
Comment (TomJKing) [inline terraform/modules/jenkins/templates/assume_role_policy.json.tpl:12]: Missing line break

File: terraform/modules/jenkins/iam.tf [hunk 2]
```diff
@@ -50,6 +50,7 @@ data "aws_iam_policy_document" "jenkins_fargate_policy_document" {
       "arn:aws:iam::${data.aws_caller_identity.current.account_id}:role/TDRJenkinsNodeLambdaRoleProd",
```
Comment (TomJKing) [inline terraform/modules/jenkins/iam.tf:50]: Has `terraform fmt` been run on this file

**Review by TomJKing — COMMENTED** [review]
Couple of minor comments

File: terraform/modules/jenkins/templates/assume_role_policy.json.tpl (hunk 1 above)
Comment (MancunianSam) [reply terraform/modules/jenkins/templates/assume_role_policy.json.tpl]: I've added this back in.

File: terraform/modules/jenkins/iam.tf (hunk 2 above)
Comment (MancunianSam) [reply terraform/modules/jenkins/iam.tf:50]: No, I keep forgetting :cry: I've run it now.

**Review by TomJKing — APPROVED** [review]

**Pull request merged**